include/deal.II/lac/petsc_matrix_base.h

```diff
@@ -975,7 +975,7 @@ namespace PETScWrappers
      */
     void mmult (MatrixBase       &C,
                 const MatrixBase &B,
-                const VectorBase &V = VectorBase()) const;
+                const VectorBase &V) const;
```

Comment:
I know that I am late to the party, but I would like to check to make sure I understand what is going on here. This function won't work in the same way it used to if we require a vector here: what is wrong with leaving the default argument in?

Reply:
This function is implemented as a protected member in `MatrixBase` which will not be called directly. The `SparseMatrix::mmult` and `MPI::SparseMatrix::mmult` implement their own default arguments.